src/core/SplashScreen.cpp

```diff
@@ -83,6 +82,13 @@ void GOSplash::DrawText(wxBitmap& bitmap)
 	wxMemoryDC dc(bitmap);
 	wxFont font;
 
+    #ifdef ASIO_INCLUDED
+        /* Set ASIO license image */
+        wxImage asio = GetImage_ASIO();
+        wxBitmap m_asio = (wxBitmap)asio.Scale(asio.GetWidth() * 0.1, asio.GetHeight() * 0.1, wxIMAGE_QUALITY_HIGH);
+        dc.DrawBitmap(m_asio, 70, 10);
+    #endif
+
```

Comment:
1. Please move the image down. You can get this code from my PR
2. I've splitted the line 114 to the non-asio and the asio-related parts in my PR, so the asio copyright is displayed only on windows. Please do the same here.